### PR TITLE
Show attractor on the visualization

### PR DESCRIPTION
**Pull request by xtuc**

### CODE REVIEW

File: server/swarmstate.go [hunk 1]
```diff
@@ -3,14 +3,15 @@ package server
...
+	"math/rand"
 
 	"github.com/netgusto/bytearena/utils"
 	uuid "github.com/satori/go.uuid"
 	"github.com/scryner/lfreequeue"
 )
 
 type SwarmState struct {
-	pin              *utils.Vector2
+	Pin              *utils.Vector2
```
Comment (xtuc) [inline server/swarmstate.go:14]: I made `Pin` public to be able to access it from the visualization update

File: cmd/arena/main.go [hunk 2]
```diff
@@ -89,10 +89,18 @@ func wsendpoint(w http.ResponseWriter, r *http.Request, stateChan chan server.Sw
...
+						Kind: "agent",
 					})
 				}
 
+				x, y := swarmstate.Pin.Get()
+
+				msg.Agents = append(msg.Agents, vizagentmessage{
+					X:	 x,
+					Y:	 y,
+					Kind: "attractor",
```
Comment (xtuc) [inline cmd/arena/main.go:101]: From the visualization point of vue, the attractor is a regular agent of kind `attractor.

Will require a refactor of this, everything is not a simple agent.

Comment (netgusto) [reply cmd/arena/main.go:101]: Yes, but this will do for now.

**Comment by netgusto** [comment]
LGTM.

**Comment by edi9999** [comment]
LGTM too.

**Pull request merged**